pkg/scheduler/cache/cache.go

```diff
@@ -1497,9 +1499,13 @@ func (sc *SchedulerCache) RecordJobStatusEvent(job *schedulingapi.JobInfo, updat
 			if len(msg) == 0 {
 				msg = baseErrorMessage
 			}
-			if err := sc.taskUnschedulable(taskInfo, reason, msg, nominatedNodeName); err != nil {
-				klog.ErrorS(err, "Failed to update unschedulable task status", "task", klog.KRef(taskInfo.Namespace, taskInfo.Name),
-					"reason", reason, "message", msg)
+			ts, exist := schedulingutil.GetPodStatusLastSetCache(job.UID, taskInfo.UID)
+			if !exist || nowTs-ts > 60 {
```

Comment:
I remember that we discussed to set the interval that can be configured or using backoff. Direct set `60` here is 
too empirical.